lunabot_config/config/global_costmap_sim.yml

```diff
@@ -7,6 +7,6 @@ robot_base_frame: "base_link"
 footprint: "[ [0.54615446154,0.295590],[0.546154,-0.295590],[-0.546154,-0.295590],[-0.546154,0.295590] ]"
 map_topic: "/projected_map"
 inflation: {
-  inflation_radius: 0.4,
-  cost_scaling_factor: 10,
+  inflation_radius: 0.6,
```

Comment:
Why are these parameters changed? Just for testing or are these actually better?

Reply:
These are better as right now, without it, the robot gets too close to obstacles